src/library_gl.js

```diff
@@ -75,6 +75,15 @@ var LibraryGL = {
       for (var i = 0; i < GL.MINI_TEMP_BUFFER_SIZE; i++) {
         GL.miniTempBufferViews[i] = GL.miniTempBuffer.subarray(0, i+1);
       }
+
+#if USE_WEBGL2
+      // For glInvalidateFramebuffer and glInvalidateSubFramebuffer, create a set of short fixed-length arrays to avoid
+      // having to generate any garbage in those functions.
+      GL.tempFixedLengthArray = [];
```

Comment:
better to declare this on the object, `tempFixedLengthArray: [],` , that way it isn't a new property at this time, better for JS optimizations.